### PR TITLE
Added Manual Steps for AWS SES Setup

### PR DESCRIPTION
**Pull request by TejasRGitHub**
### Feature or Bugfix
- Feature ( Documentation ) 

### Detail

When enabling email notification on share workflow, an SES identity is created on AWS SES. This PR contains documentation about the manual steps that need to be taken towards setting up SES and also additional steps to setup monitoring.  

### Relates
- https://github.com/awslabs/aws-dataall/issues/734
- https://github.com/awslabs/aws-dataall/issues/785

### Security
Please answer the questions below briefly where applicable, or write `N/A`. Based on
[OWASP 10](https://owasp.org/Top10/en/).

- Does this PR introduce or modify any input fields or queries - this includes
fetching data from storage outside the application (e.g. a database, an S3 bucket)? N/A
  - Is the input sanitized?
  - What precautions are you taking before deserializing the data you consume?
  - Is injection prevented by parametrizing queries?
  - Have you ensured no `eval` or similar functions are used?
- Does this PR introduce any functionality or component that requires authorization? N/A
  - How have you ensured it respects the existing AuthN/AuthZ mechanisms?
  - Are you logging failed auth attempts?
- Are you using or adding any cryptographic features? N/A
  - Do you use a standard proven implementations?
  - Are the used keys controlled by the customer? Where are they stored? 
- Are you introducing any new policies/roles/users? N/A
  - Have you used the least-privilege principle? How?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pages/deploy/deploy_aws.md [hunk 1]
```diff
@@ -518,6 +518,25 @@ diagram.
 With this pipeline we can now deploy the infrastructure to the deployment account(s). Navigate to AWS CodePipeline
 in the tooling account and check the status of your pipeline.
 
+## 12. Setting SES for Email Notifications
+
+Please follow instructions from below if you have enabled email notification by switching the email.active config to true in the `share_notifications` feature under `datasets` module.
```
Comment (dlpzx) [inline pages/deploy/deploy_aws.md:523]: Can we add here that we are referring to the `config.json`? Deployment is quite confusing for new data.all-ers so I would make it as clear as possible

File: pages/deploy/deploy_aws.md [hunk 2]
```diff
@@ -518,6 +518,25 @@ diagram.
...
+send email notifications via SES, but your AWS account is in the Sandbox mode. When in Sandbox mode, you will have to verify each 
+recipient email id manually. In order to get your SES account out of Sandbox, please follow the instructions in <a href="https://docs.aws.amazon.com/ses/latest/dg/request-production-access.html">Moving out of Sandbox</a> .
+Once your AWS SES account is out of sandbox you can send email to any recipient email id without any prior verification. 
+
+### Monitoring for Email Bounces
+When SES Stack is deployed during the pipeline stage, it will setup a <a href="https://docs.aws.amazon.com/ses/latest/dg/using-configuration-sets.html">configuration set</a>
+which will send any email bounces, delivery failures , rejects & complaints to an SNS topic. In this step, you can add subscriptions to this SNS topic to monitor email delivery problems
+
+In order to do that goto AWS Console -> SNS -> Select the SNS topic which would look like `{resource_prefix}-{envname}-SNS-Email-Bounce-Topic` ( where resource_prefix and envname are specified in the cdk.json ) -> Create Subscription. You can attach multiple subscriptions to
```
Comment (dlpzx) [inline pages/deploy/deploy_aws.md:536]: typo: "goto" and extra space before comma "failures , rejects"

**Review by dlpzx — COMMENTED** [review]
@TejasRGitHub ultra minor comments. I like it, it is very concise and clear

File: pages/deploy/deploy_aws.md (hunk 1 above)
Comment (TejasRGitHub) [reply pages/deploy/deploy_aws.md]: Yes. I made changes to specify where exactly the configuration is present.

File: pages/deploy/deploy_aws.md (hunk 2 above)
Comment (TejasRGitHub) [reply pages/deploy/deploy_aws.md]: Addressed typos in the next update

**Review by noah-paige — APPROVED** [review]
Followed the below docs to set up SES email notification - looks good

**Comment by noah-paige** [comment]
Thanks @TejasRGitHub

**Pull request merged**